sensor_msgs/Image depth_image
sensor_msgs/Image rgb_image
---
bool success
ggcnn/Grasp2D best_grasp
